pages/bib/[id].tsx

```diff
@@ -49,8 +111,50 @@ export default function BibPage({
       <Layout isAuthenticated={isAuthenticated} activePage="bib">
         <Heading level="h2">{bib.title}</Heading>
         <BibDetails key="top-details" details={topDetails} />
-        {bib.itemTableData ? (
-          <ItemTable itemTableData={bib.itemTableData} />
+        {bib.showItemTable ? (
+          <>
+            <Heading
+              data-testid="item-table-heading"
+              level="h3"
+              size="heading4"
+              mb={{ base: "s", md: "m" }}
+            >
+              Items in the library and off-site
+            </Heading>
+            <Box id="item-table" ref={itemTableScrollRef}>
+              {itemsLoading ? (
+                <SkeletonLoader showImage={false} />
+              ) : itemFetchError ? (
+                <Banner
+                  type="negative"
+                  content="There was an error fetching items. Please try again with a different query."
+                />
+              ) : (
+                <>
+                  <Heading
+                    data-testid="item-table-displaying-text"
+                    level="h4"
+                    size="heading5"
+                    mb={{ base: "s", md: "m" }}
+                  >
+                    {buildItemTableDisplayingString(
+                      itemPage,
+                      bib.numPhysicalItems
```

Comment:
The text `Displaying 1-20 of 816 items` never updates on the client side. So if you keep pressing "next" in the pagination, the "1-20" never updates. If you go to a page with `item_page=X` directly, it'll work. 

This is because it's reading `itemPage` from `getServerSideProps` which is ideal for getting the data from the server on first load. But on clientside interaction, the value gets updated appropriately from `query.item_page` (from the `useRouter()` hook). So I think you need to find a way to get `itemPage` on first load only and then update the value based on user interaction the Pagination from `handlePageChange` function.

Reply:
Good catch. This was happening because the page number wasn't being updated in state. Just added a useState that fixed the issue. Could also access the param via the router query but it would require another parseInt.